srcpkgs/babashka/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'babashka'
+pkgname=babashka
+version=0.0.88
+revision=2
+archs="x86_64"
+build_style=fetch
+hostmakedepends="unzip"
+short_desc="Native fast-starting Clojure scripting environment"
+maintainer="Patrick Pichler <mail@patrickpichler.dev>"
+license="GPL-3.0-or-later"
```

Comment:
Wrong license, the project states EPL-1.0.

Reply:
Thank you for pointing out! Should be fixed new